### DISCUSSION TITLE
Change QSL entries from "electronic" to "eQSL"

### DISCUSSION THREAD

**Discussion by lbismarck** [magicbug/Cloudlog#2545] [General]
Hi all,

I made an operative mistake, marking all QSOs for which I received eQSL cards as "Electronic" instead of using the eQSL fields.
The problem is that I am preparing my logbook for DXCC and I need to know/filter which important QSOs (countries) have been officially confirmed or not (means only direct, bureau, Manager & of course Lotw). Then I would export these QSOs and either upload them to Lotw for approval or start a different QSL process. Right now some QSOs appear as "confirmed" eventhough I only have eQSL which does not count.

Hence I would like to do a batch operation where I transfer the content of all the "Electronic" fields (sent and received) that currently have content over to their corresponding eQSL fields (send and received).

Alternatively I could maybe also batch-remove all entries from "electronic" QSL fields and then do a sync (download all confirmations) with eQSL?

Any help with this would be highly appreciated.
Thanks,
Leo

**Comment by magicbug** [comment]
Theres no frontend tool that will allow you todo this, you can use Advanced Search (click on the search button) to get to it, but it doesn't allow match operations and I have a feeling its beyond the functionality of Adavnced Logbook at the minute.

**Reply by lbismarck** [reply]
yes that's what I was thinking as well and I am guessing there is no simple command line script that would switch back all database values where "electronic" (sent and received) is NOT NULL back to NULL or "method" I believe is the standard place holder?

**Reply by AndreasK79** [reply]
If you are able to run SQL queries, it should be doable to fix this.
